test/fixtures/out/with-codespans.astro

```diff
@@ -2,16 +2,16 @@
 let hello = "hello";
 let space = " ";
 let helloWorld = `${hello + space}world`;
-import Markdown from "astro/components";
+import MarkdownComponent from "astro/components/Markdown.astro";
 ---
-
 <div>{`${helloWorld + space}too many template strings`}</div>
-<Markdown>
+<MarkdownComponent>
   `hello i am a codespan`
 
   ```js
   let story = "hello i am a codefence";
 
   story.length;
   ```
-</Markdown>
+
```

Comment:
I got Markdown formatting working but now it inserts a blank line at the end 😭 

I think this is because Prettier is parsing it with its own formatter, and inserts a final end line? Unsure.